effekt/jvm/src/test/scala/effekt/LLVMTests.scala

```diff
@@ -52,5 +53,22 @@ class LLVMTests extends EffektTests {
     examplesDir / "pos" / "issue733.effekt",
   )
 
-  override lazy val ignored: List[File] = missingFeatures
+  override lazy val ignored: List[File] = missingFeatures ++ noValgrind(examplesDir)
+}
+
+/**
+ * Documentation of tests that succeed in running, but fail valgrind
+ * and their reason
+ */
+def noValgrind(examplesDir: File): List[File] = List(
+  examplesDir / "llvm" / "prompt-duplication.effekt",
+)
+
+class LLVMNoValgrindTests extends EffektTests {
+  def backendName = "llvm"
+
+  override def valgrind = false
+  override def debug = false
+
+  override lazy val positives: List[File] = noValgrind(examplesDir)
 }
```

Comment:
I don't really like this, but I don't have any other quick solutions for this...

Reply:
We could just have a convention of `llvm/novalgrind/...` or something, but that doesn't scale too well for StdlibLLVMTests...